assets/ajax-reload.js

```diff
@@ -0,0 +1,65 @@
+window.HasteAjax = {
+    /**
+     * Requests in progress
+     */
+    inProgress: {},
+
+    /**
+     * Fire the event
+     * @param {String} event
+     */
+    fireEvent: function (event) {
+        var els = {};
+
+        Array.from(document.querySelectorAll('[data-haste-ajax-listeners]')).forEach(function (el) {
+            if (el.dataset.hasteAjaxListeners.split(' ').indexOf(event) !== -1) {
+                els[el.dataset.hasteAjaxId] = el;
+            }
+        });
+
+        if (Object.keys(els).length < 1) {
+            console.warn('There are no eligible elements for event "' + event + '"');
+            return;
+        }
+
```

Comment:
`els` should be filtered so the same element is not reloaded multiple times (which may happen when they listen to multiple events and they are all fired)